### PR TITLE
Update 01-setting-up.md

### PR DESCRIPTION
**Pull request by bitsuns**
Install dependencies first

### CODE REVIEW

File: lessons/01-setting-up.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ up our project.
 ```
 git clone <tutorial url>
 cd react-router-tutorial
+npm install
 git checkout start
```
Comment (gaearon) [inline lessons/01-setting-up.md]: `npm install` should probably come after `git checkout` so it is performed on the correct branch?

Comment (bitsons) [inline lessons/01-setting-up.md]: Good catch. Thank you.

**Comment by bitsons** [comment]
There's over a dozen PR's for this:
``` 
is:pr is:open "npm install"
```

**Comment by ryanflorence** [comment]
We changed the repo structure to make way more sense, previously the git history was part of the "structure" and that was a terrible idea, making PRs impossible to merge.

Now each lesson has its own directory so we can just click the merge button!

Mind updating your PR to work with the current code on the **master** branch? Thanks!

**Comment by bitsuns** [comment]
Understood.

This PR isn't needed; already incorporated into latest revision.

**Pull request closed without merging**